source/detail/implementations/cell_impl.hpp

```diff
@@ -61,5 +61,21 @@ struct cell_impl
     optional<comment *> comment_;
 };
 
+inline bool operator==(const cell_impl &lhs, const cell_impl &rhs)
```

Comment:
Currently this never gets called because cell is only comparing its internal pointers for equality (reference equality, not value equality). This is why the operator implementation isn't yet exhaustive